Decred Wallet/Features/Navigation Menu/NavigationMenuBaseController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  NavigationMenuBaseController.swift
+//  Decred Wallet
+//
+//  Created by Sprinthub on 11/09/2019.
+//  Copyright © 2019 Decred. All rights reserved.
+//
+
+
+import UIKit
+
+
+class NavigationMenuBaseController: TabMenuController {
```

Comment:
2 custom classes should not be necessary to subclass `UITabBarController`, please merge `TabMenuController` and `NavigationMenuBaseController`

Reply:
resolved in 66727f1